### DISCUSSION TITLE
Still not able to send otp

### DISCUSSION THREAD

**Discussion by Kushwaha2021** [saran-surya/email_auth#33] [Q&A]
email-auth >> Failed to send OTP ❌
I/flutter (27574): email-auth >> Message from server :: null

**Comment by saran-surya** [comment]
hi @Kushwaha2021 , can we have the discussions in one QA as it becomes difficult to map and address all issues.

Also seems like you have not yet setup the server, Kindly share the complete log. And a screenshot if possible.
